.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
please don't commit local files, search for how to not commit files that you don't need but keep them in your local repo